enterprise/api/v1/serializers.py

```diff
@@ -952,3 +952,39 @@ def validate(self, data):  # pylint: disable=arguments-differ
         if not data.get('email') and not data.get('email_csv'):
             raise serializers.ValidationError('Must include either email or email_csv in request.')
         return data
+
+
+# pylint: disable=abstract-method
+class EnterpriseCustomerBulkSubscriptionEnrollmentsSerializer(serializers.Serializer):
+    """
+    Serializes a licenses info field for bulk enrollment requests.
+    """
+    licenses_info = serializers.ListField(
+        child=serializers.DictField(
+            child=serializers.CharField(required=False),
+        ),
+        required=False,
+    )
+    reason = serializers.CharField(required=False)
+    salesforce_id = serializers.CharField(required=False)
+    discount = serializers.DecimalField(None, 5, required=False)
+    notify = serializers.BooleanField(default=False)
+
+    def create(self, validated_data):
+        return validated_data
+
+    def validate(self, data):  # pylint: disable=arguments-differ
+        if not data.get('licenses_info'):
```

Comment:
why do you have to do this here? Why can you not make required=False at line 966 instead?

Reply:
I was following the patterns in the other bulk enrollment endpoint. Happy to make the change if you think it's worth it.

Reply:
Please go ahead and change it and make sure there's a test to cover your validation if it isn't there yet.

Reply:
Could you elaborate please? Are you saying I should keep the way in which we were validating the serializer?

Reply:
Yes.

Reply:
I did some research and while the required=True thing might work, the 'list of objects' validation requires more finagling then it might actually be worth to make it happen automagically.